### DISCUSSION TITLE
Adding the other principle sections

### DISCUSSION THREAD

**Discussion by RobJohnsonHO** [UKHomeOffice-test/docs-as-code-nunjucks#23] [Announcements]
Trying out the announcement feature discussion.

I will start a new feature branch for the software design top level principles.  I'll try creating issues off this discussion and see how the flow works.  

We can drop updates in discussion and any problems/issues etc...

**Comment by RobJohnsonHO** [comment]
@jeff-horton-ho-sas @edhamiltonHO @robertdeniszczyc2 
What do you think about the project structure like so:

root
    - _includes
    - _site
    - assets
    - principles
        - software design
        - ways of working
        - (nn)....

This way we can roll up the individual principle segments into a top level one at a later stage?

**Reply by RobJohnsonHO** [reply]
Weird, software design and ways of working in my preview show as indented but not when its posted.

I'm suggesting guild areas as folders under the principles top folder.

**Reply by edhamiltonHO** [reply]
I'm not sure if there should be subfolders under the principle top folder, principles often cut across the different guilds. I think we could use a flat space inside the principle folder and just use tags within those items to relate them to associated areas/guilds etc and make searchable. I think this fits the nature of the principle as an object, and would be manageable if we don't have too many principles, but we wil want to mark ownership/stewardship of a principle somehow

**Reply by RobJohnsonHO** [reply]
ok cool, if you could merge your changes for the spelling change then I can get cracking :)

**Reply by edhamiltonHO** [reply]
👍 Have created PR #26
